core/sql/executor/HiveClient_JNI.cpp

```diff
@@ -42,17 +43,16 @@ static const char* const hvcErrorEnumStr[] =
  ,"Java exception in close()."
```

Comment:
Should there also be exception text for ,HVC_ERROR_INIT_EXCEPTION?

Reply:
Yes. Fixed it